src/machine/board_nano-33-ble.go

```diff
@@ -0,0 +1,80 @@
+// +build nano_33_ble
+
+package machine
+
+const HasLowFrequencyCrystal = false
```

Comment:
I don't know if this board actually has it.
It works with this value, probably will work even better if enabled, I just don't know how to test.

Reply:
If it works with `HasLowFrequencyCrystal = true`, there is probably a low frequency oscillator. Most boards (except for the smallest/cheapest) have one.